.gitignore

```diff
@@ -1,8 +1,8 @@
 .idea/
 .vscode/
 *.iml
-.env
 .DS_Store
 venv/
 .pytest_cache/
 **__pycache__/
+.env
```

Comment:
The `.env` file is correctly included in the `.gitignore` file to prevent sensitive information from being tracked. No changes are needed for this line.